packages/admin/admin/src/common/toolbar/Toolbar.tsx

```diff
@@ -55,19 +83,40 @@ const MainContentContainer = createComponentSlot("div")<ToolbarClassKey>({
     flex: 1;
 `);
 
+const Breadcrumbs = createComponentSlot(ToolbarBreadcrumbs)<ToolbarClassKey>({
+    componentName: "Toolbar",
+    slotName: "breadcrumbs",
+})();
+
 export const Toolbar = (inProps: ToolbarProps) => {
-    const { children, elevation = 1, slotProps, ...restProps } = useThemeProps({ props: inProps, name: "CometAdminToolbar" });
+    const {
+        children,
+        hideTopBar = false,
```

Comment:
I think the default value is unnecessary here.

```suggestion
        hideTopBar,
```

Reply:
I prefer it to be explicit